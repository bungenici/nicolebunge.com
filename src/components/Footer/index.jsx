import PropTypes from 'prop-types';
import React, { PureComponent } from 'react';
import './style.css';

class Footer extends PureComponent {
  static propTypes = {
    children: PropTypes.node,
  };

  static defaultProps = {
    children: null,
  };

  render() {
    const { children } = this.props;

    return (
      <footer
        className="footer"
        role="contentinfo"
      >
        {children}
      </footer>
    );
  }
}

export default Footer;
