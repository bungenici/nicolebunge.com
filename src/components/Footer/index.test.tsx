/* eslint-disable import/no-extraneous-dependencies */

import { render } from '@testing-library/react';
import React from 'react';
import Footer from '.';

describe('Footer component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
