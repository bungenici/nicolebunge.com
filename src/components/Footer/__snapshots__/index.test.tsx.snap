// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
    role="contentinfo"
  />
</DocumentFragment>
`;
